module/migrations/actor-migrations-manager.js

```diff
@@ -1,29 +1,39 @@
-
+import { PrimeTables } from "../prime_tables.js";
 export class ActorMigrationsManager {
 
     static async assessMigrationRequirements()
     {
         const unclonedV1Characters = this.getUnclonedV1Characters();
         if (unclonedV1Characters.length === 0)
         {
-            this.removeV2Clones();
+            ui.notifications.info("Culling the clones");
+            await this.removeV2Clones();
         }
         else
         {
+            ui.notifications.info("Creating V2 character clones");
             const newV2Clones = this.cloneAndUpgradeToV2(unclonedV1Characters);
-            const newV2Characters = await this.createNewCharacters(newV2Clones);
+            this.createNewCharacters(newV2Clones);
```

Comment:
Has this intentionally been made synchronous?

Reply:
Yeah, the async stuff is inside the method. Could return the Promise.all instead and chain from it here, but there didn't seem much point? Especially given the actors aren't even fully created by the end of the process - they can't have child items until they're fully created.

I'd like to refactor how the stat items are created, but I'm trying to avoid doing more work than is required to get us to a launch point, so it's a balance of getting it working / paying tech. debt where it makes sense.

Reply:
Cool, just thought it was worth checking 👍